src/Forms/NocaptchaField.php

```diff
@@ -180,44 +196,65 @@ public function Field($properties=array()) {
             user_error('You must configure Nocaptcha.site_key and Nocaptcha.secret_key, you can retrieve these at https://google.com/recaptcha', E_USER_ERROR);
         }
 
-        $form = $this->getForm();
-
         if ($this->config()->get('recaptcha_version') == 2) {
-            $exemptActionsString = implode("' , '", $form->getValidationExemptActions());
+            $this->configureRequirementsForV2();
+        } else {
+            $this->configureRequirementsForV3();
+        }
 
+        return parent::Field($properties);
+    }
+
+    private function configureRequirementsForV2()
```

Comment:
Can you throw together a quick php doc for this method, let's also change this to protected so it can be overridden.